api_app/script_analyzers/file_analyzers/floss.py

```diff
@@ -25,4 +36,23 @@ def run(self):
             "callback_context": {"read_result_from": fname},
         }
         req_files = {fname: binary}
-        return self._docker_run(req_data, req_files)
+        result = self._docker_run(req_data, req_files)
+        result["exceeded_max_number_of_strings"] = {}
+
+        self.url = "http://static_analyzers:4002/stringsifter"  # For Ranking
```

Comment:
define this at the very top just under the class name

Reply:
Initially, FLOSS url should be used, After getting list of strings, then URL needs to be changed. However, It can be moved to the start of `run()` function.

Reply:
why not just declare 2 different variables at the top of the class then

Reply:
`self._docker_run()` extracts url from `self.url`.

Initial Run of `self.docker_run()` is required for FLOSS strings. 

Second Run is for rank_strings. (Had to change URL before running this).